functions/notify_slack.py

```diff
@@ -247,10 +247,12 @@ def notify_slack(subject, message, region):
 
     if "source" in message and filter_message_from_slack(message):
         print("filtering message, not posting to slack")
+        print(message)
```

Comment:
Do we need these `print(message)` lines here?

Reply:
I believe @rraub mentioned wanting to log these messages so we have them showing up in Datadog if I understood correctly? If not I can remove these!

Reply:
I could go either way on this, I was an idea to at least log these messages which could be useful in the future if we want. There are other options for logging to datadog right from the cloudwatch events but I dont feel to strongly about this.